docs/csharp/programming-guide/arrays/codesnippet/CSharp/object-and-collection-initializers_7.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace ExampleProject
+{
+    class FormattedAddresses : IEnumerable<string>
+    {
+        private List<string> internalList;
+        
+        public IEnumerator<string> GetEnumerator()
+        {
+            return internalList.GetEnumerator();
+        }
+
+        IEnumerator IEnumerable.GetEnumerator()
+        {
+            return internalList.GetEnumerator();
+        }
+
+        public FormattedAddresses()
```

Comment:
I'd prefer removing the constructor and initializing `internalList` as a field initializer.